subprojects/core/src/main/java/org/gradle/plugin/use/internal/PluginUseScriptBlockMetadataCompiler.java

```diff
@@ -95,6 +114,16 @@ public void visitMethodCallExpression(MethodCallExpression call) {
 
                                 break;
                             case "version":
+                                fullExpression = hasSinglePropertyExpressionArgument(call);
+                                if (fullExpression != null) {
```

Comment:
This allows to do something like:
```
plugins {
    id("my.plugin.id") version libs.versions.myPlugin
}
```
Correct?
And then 👍

Reply:
Correct. Also because supporting `.get()` in the AST restrictor would become significantly more complex.